.eslintrc.json

```diff
@@ -20,7 +20,8 @@
         "fixStyle": "inline-type-imports"
       }
     ],
-    "@typescript-eslint/no-unused-vars": "off",
+    "@typescript-eslint/no-unused-vars": "warn",
```

Comment:
This is intentional, it's handled by the import plugin so we have to turn this one off.

```suggestion
```